src/com/intellij/idea/plugin/hybris/project/configurators/impl/DefaultLibRootsConfigurator.java

```diff
@@ -101,6 +101,7 @@ public void configure(
             hybrisModuleDescriptor.createBootstrapLib(sourceCodeRoot, modifiableModelsProvider);
         }
 
+        System.out.println(">>>" + moduleDescriptor.getClass() + " " + moduleDescriptor.getName());
```

Comment:
Is it just for debugging?

Reply:
Removed. It should not have been commited at all